python/ordinal-classifier.ipynb

```diff
@@ -0,0 +1,466 @@
+      ],
+      "text/plain": [
+       "MultiOutputClassifier(estimator=LogisticRegression(max_iter=1000), n_jobs=-1)"
```

Comment:
While it will work for this task, I don't really recommend using logistic regression here as it's a rather basic classifier. Also it's best practise to instantiate the classifier prior to using it so you would write this first `lr = LogisticRegresssion(max_iter=1000)`